src/Exception/Comment/TalkCommentDeletionException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Exception\Comment;
+
+use Exception;
+use Teapot\StatusCode\Http;
+use Throwable;
+
+final class TalkCommentDeletionException extends Exception implements Throwable
```

Comment:
The `Throwable` interface is already implemented by `Exception`, can we remove the `implements`?